lib/IDE/Refactoring.cpp

```diff
@@ -2602,6 +2602,143 @@ bool RefactoringActionLocalizeString::performChange() {
   return false;
 }
 
+void processParameters(ArrayRef<const ParameterList *> allParamsList, swift::SourceLoc startLocation, SourceEditConsumer &EditConsumer, SourceManager &SM) {
```

Comment:
We try to keep lines at most 80 characters -- can you wrap these parameters?

Reply:
This also applies to the rest of the code. LLVM style dictates 80 character line limits and 2 space indents.

If you're using Xcode for this, you can set your indent width to 2 spaces and reindent a selection with `Ctrl+I`.